app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class KeywordsController < ApplicationController
+      include ::Pagy::Backend
+      include API::V1::Pagy::JSONAPIConcern
+
+      def index
+        pagy, keywords_list = pagy(keywords)
+
+        if keywords.any?
+          render json: KeywordSerializer.new(keywords_list, pagy_options(pagy)).serializable_hash, status: :ok
+        else
+          render_empty
+        end
+      end
+
+      private
+
+      def render_empty
```

Comment:
How about moving it to `API::V1::ApplicationController`? then all child controllers can use it, maybe the method can access the message parameter also, so childs controller can pass message

```ruby
def render_empty(message)
    render json: { meta: message), data: [] }, status: :ok
end
```

Reply:
Moved with a meta_message params ;-)